.eslint-doc-generatorrc.js

```diff
@@ -0,0 +1,20 @@
+/** @type {import('eslint-doc-generator/dist/lib/options').GenerateOptions} */
```

Comment:
Nice! I'll export this type in the future: https://github.com/bmish/eslint-doc-generator/issues/261

Reply:
FYI the public type is now available:

* https://github.com/bmish/eslint-doc-generator/releases/tag/v0.26.0
* https://github.com/bmish/eslint-doc-generator#configuration-file
* https://github.com/bmish/eslint-doc-generator/pull/264

Reply:
sweet - I'm going to leave it as-is for now but will address when https://github.com/bmish/eslint-doc-generator/issues/260 is done

Reply:
Okay well you might want to switch to the public type sooner since:
* The type no longer exists at the location you're using it--would cause an issue with any upgrade
* I don't think I'll be able to work on the TypeScript config support any time soon

Reply:
Looks like this is now broken for you after the update.

Reply:
Nope nothing to do with this because

> Currently we don't actually check JS files with TS

🙂

Reply:
Regardless of that, I find it valuable for the hover-over documentation / auto-complete too.

Reply:
As do I, but that's primarily useful when you edit the config file so I'll either update the type then or when I setup type checking for JS files.

Reply:
Sounds good!